raiden/network/proxies/token_network.py

```diff
@@ -52,6 +52,17 @@
 
 log = structlog.get_logger(__name__)  # pylint: disable=invalid-name
 
+ParticipantDetails = Dict[
```

Comment:
Shouldn't this kind of type-alias be in `raiden.utils.typing`?

Reply:
no. not all type aliases need to be in typing if they are local to the module they should stay in the module.

Also note: This PR is **NOT** ready for review. I am pushing only when I wanna move stuff from desktop to laptop or vice versa and when I want to run travis.